libs/designsystem/button/src/button.component.scss

```
@@ -30,7 +30,10 @@ $button-margin: utils.size('xxxs');
   // See: https://sass-lang.com/documentation/breaking-changes/mixed-decls/
   & {
     font-size: utils.font-size('xs');
-    height: $button-xs-icon-only-size;
+
+    // use fixed size unless font-size grows too large, then let button
+    // height adjust to font-size (effectively when text is resized to 150% and above)
+    height: max($button-xs-icon-only-size, utils.relative-spacing('s'));
```

Whats the strategy for sizing here? With this code it will not grow the first 150% font scaling. Is this what we want? Could also be that it grows the first 150% and then not grow further after that. Below is just showing the alternative
```suggestion
    height: min(utils.size('l'), utils.relative-spacing('m'));
```

---

I think it might be odd for users who only increase the font-size a little. Here they will see some components grow while others will stay the same size. Wouldn't it be better if all components grow when scaling, but are capped at a certain point.

---

The strategy is to zoom the text as requested by the user, but only start scaling the height of the button when it becomes cramped, otherwise let the button remain fixed size because we want to leave as much room as possible to text to get a more information dense UI.